src/Microsoft.SqlTools.ServiceLayer/LanguageServices/PeekDefinition.cs

```diff
@@ -61,7 +62,7 @@ internal PeekDefinition(ServerConnection serverConnection, ConnectionInfo connIn
             Initialize();
         }
 
-        private Database Database
+        internal Database Database
```

Comment:
Since the check for a new `Database` field only occurs if `this.database == null`, Is it possible for the following scenario to occur?
- `Database` getter is accessed, setting `this.database` to not null and `Database` field to `master`
- `use tempdb` is called, changing the `Database` field for all open connections to `tempdb`
- `Database` getter is accessed again, returning the non-null `this.database` with `Database = master` instead of `Database = tempdb`.